### PR TITLE
Fix webui for agent

### PR DESCRIPTION
**Pull request by Itxaka**
It was mistakenly added to the env steps but it should be run in the steps instead

Fixes: https://github.com/kairos-io/kairos/issues/2520

### CODE REVIEW

**Review by mauromorales — APPROVED** [review]

File: packages/system/kairos-agent/collection.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 packages:
   - name: "kairos-agent"
     category: "system"
-    version: "2.10.2"
+    version: "2.10.2+1"
```
Comment (mauromorales) [inline packages/system/kairos-agent/collection.yaml:4]:
```suggestion
    version: "2.10.3"
```

File: packages/system/kairos-agent/collection.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ packages:
     description: "Lifecycle agent for kairos"
   - name: "kairos-agent"
     category: "fips"
-    version: "2.10.2"
+    version: "2.10.2+1"
```
Comment (mauromorales) [inline packages/system/kairos-agent/collection.yaml:15]:
```suggestion
    version: "2.10.3"
```

**Comment by mauromorales** [comment]
let's use this PR to also bump the agent https://github.com/kairos-io/packages/pull/844

File: packages/system/kairos-agent/collection.yaml (hunk 2 above)
Comment (Itxaka) [reply packages/system/kairos-agent/collection.yaml]: not really no? version is the fixed one from agent, this is a revision of the package only, nothing changed upstream in the agent

Comment (Itxaka) [reply packages/system/kairos-agent/collection.yaml]: crap, didnt see your comment below loool

**Pull request merged**